examples/hosts/app-integration/external-controller/src/app.ts

```diff
@@ -4,37 +4,12 @@
  */
 import { SharedMap } from "@fluid-experimental/fluid-framework";
 import {
-    FrsAzFunctionTokenProvider,
-    FrsClient,
-    FrsConnectionConfig,
-    InsecureTokenProvider,
-} from "@fluid-experimental/frs-client";
-import { generateUser } from "@fluidframework/server-services-client";
+ TinyliciousClient,
+} from "@fluid-experimental/tinylicious-client";
 import { DiceRollerController } from "./controller";
-import { ConsoleLogger } from "./ConsoleLogger";
 import { renderAudience, renderDiceRoller } from "./view";
 
 // Define the server we will be using and initialize Fluid
-const useFrs = process.env.FLUID_CLIENT === "frs";
```

Comment:
This can all go back before merging, I just needed a demo with only using Tinylicous.

Reply:
I think we need to have one for frs and one for tinylicious

Reply:
yeah, i wanted to make sure we liked this approach in tinylicious, and then i'll port to FRS and set up proper examples